docs/creating_integration_manifest.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Integration Manifest"
+sidebar_label: "Manifest"
+---
+
+Each integration has a manifest file to specify basic information about an integration. This file is stored as `manifest.json` in your integration directory. It is required to add such a file, including for custom components.
+
+```json
+{
+  "domain": "hue",
+  "name": "Philips Hue",
+  "documentation": "https://www.home-assistant.io/components/hue",
+  "dependencies": ["mqtt"],
+  "codeowners": ["@balloob"],
+  "requirements": ["aiohue==1.9.1"]
+}
+```
+
+Or a minimal example that you can copy into your project:
+
+```json
+{
+  "domain": "your_domain_name",
+  "name": "Your Intgration",
+  "documentation": "https://www.example.com",
+  "dependencies": [],
+  "codeowners": [],
+  "requirements": []
+}
+```
+
+## Domain
+
+The domain is a short name consisting of characters and underscores. This domain has to be unique and cannot be changed. Example of the domain for the mobile app integration: `mobile_app`.
+
+## Name
+
+The name of the integration.
+
+## Documentation
+
+Website that contains documentation on how to use your integration. If this integration is being submitted for inclusion in Home Assistant, it should be `https://www.home-assistant.io/components/<domain>`
+
+## Dependencies
+
+Dependencies are other Home Assistant integrations that you want Home Assistant to set up successfully prior to the integration being loaded. This can be necessary in case you want to offer functionality from that other integration, like using webhooks or an MQTT connection.
+
+## Code Owners
+
+GitHub usernames or team names of people that are responsible for this integration. You should add at least your GitHub username here, as well as anyone who helped you to write code that is being included.
+
+## Requirements
+
+Requirements are Python libraries or modules that you would normally install using `pip` for your component. Home Assistant will try to install the requirements into the `deps` subdirectory of the Home Assistant [configuration directory](https://www.home-assistant.io/docs/configuration/) if you are not using a `venv` or in something like `path/to/venv/lib/python3.6/site-packages` if you running in a virtual environment. This will make sure that all requirements are present at startup. If steps fails like missing packages for the compilation of a module or other install errors, the component will fail to load.
```

Comment:
```suggestion
Requirements are Python libraries or modules that you would normally install using `pip` for your component. Home Assistant will try to install the requirements into the `deps` subdirectory of the Home Assistant [configuration directory](https://www.home-assistant.io/docs/configuration/) if you are not using a `venv` or in something like `path/to/venv/lib/python3.6/site-packages` if you running in a virtual environment. This will make sure that all requirements are present at startup. If steps fail, like missing packages for the compilation of a module or other install errors, the component will fail to load.
```